app/forms/gobierto_admin/gobierto_participation/process_form.rb

```diff
@@ -27,6 +28,14 @@ class ProcessForm
 
       validates :site, :title_translations, :process_type, presence: true
       validates :process_type, inclusion: { in: ::GobiertoParticipation::Process.process_types }
+      validates_presence_of :starts, :ends, if: :process?
```

Comment:
I wouldn't add these validations over `starts` and `ends`, since they were removed on purpose [here](https://github.com/PopulateTools/gobierto/pull/973) to make the process creation flow less restrictive. Applies to both processes and groups.

Please check with @furilo or @ferblape.

Reply:
Yes, those validations should be removed